storage_service/locations/models/package.py

```diff
@@ -589,6 +714,12 @@ def backlog_transfer(self, origin_location, origin_path):
             source_path=os.path.join(self.origin_location.relative_path, self.origin_path),
             destination_path=self.current_path,  # This should include Location.path
             destination_space=dest_space)
+
+        try:
+            self._index_file_data_from_transfer_mets(prefix=os.path.join(dest_space.staging_path, self.current_path))  # create File entries for every file in the transfer
+        except StorageException as e:
+            LOGGER.warning("Transfer METS data could not be read: %s", str(e))
```

Comment:
Yeah, I'm a little torn on that one. You can still fetch it, just not necessarily in a useful or straightforward way.